velox/docs/functions/spark/datetime.rst

```diff
@@ -7,6 +7,10 @@ Convenience Extraction Functions
 
 These functions support TIMESTAMP and DATE input types.
 
+.. spark:function:: months_between(timestamp, timestamp, boolean) -> double
```

Comment:
Please, move this function to preserve alphabetical order.

Reply:
@mbasmanova done.